src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -480,5 +485,16 @@ private void FixEncoding(ProjectInfo projectInfo, string globalSourceEncoding)
                 }
             }
         }
+
+        private static bool IsInNuGetCache(string filePath)
+        {
+            // List of nuget cache paths in Windows and Unix:
+            // https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders
+            var nugetCachePath =  PlatformHelper.IsWindows() ?
+                Environment.ExpandEnvironmentVariables(@"%userprofile%\.nuget\packages") :
+                Environment.ExpandEnvironmentVariables(@"%HOME%/.nuget/packages");
+
+            return filePath.Contains(nugetCachePath);
```

Comment:
or could just check `.nuget/packages` based on [this comment](https://github.com/SonarSource/sonar-scanner-msbuild/issues/1032#issuecomment-1141865283)? because this is called when outside of base dir path